docs/user-guide/work-packages/share-work-packages/README.md

```diff
@@ -72,11 +76,13 @@ You can also remove the user from the list by clicking on **Remove** next to the
 
 For an overview of all work packages that have been shared with other users or groups, navigate to the [global modules](../../home/global-modules/), select the module **Work Packages** and choose the filter **Shared with users** from the list of default work package filters on the left side. 
 
+The default view will include the **Shared with** column, which lists all users and/or groups that a particular work package was shared with. If there are more than two shared users and/or groups, two of them will be listed and the overall number will be indicated in the badge. Clicking on that number will show the details.
```

Comment:
@MayaBerd, don't know if this information is correct 100%, I think we don't control the amount of users displayed in the column itself rather than display as much as we can with the available column width. @oliverguenther can you confirm this?

Reply:
It will render at max 2 values, it's correct 👍